.mailmap

```diff
@@ -767,10 +903,13 @@ Yuri Weinstein <yuriw@redhat.com> <yweinste@redhat.com>
 Yuri Weinstein <yuriw@redhat.com> <yweinstei@redhat.com>
 Yuri Weinstein <yuriw@redhat.com> <yweinstein@redhat.com>
 Yuval Lifshitz <ylifshit@redhat.com> <yuvalif@yahoo.com>
-Zac Dover <zac.dover@gmail.com> <1445107+zdover23@users.noreply.github.com>
+Zac Dover <zac.dover@proton.me>
+Zac Dover <zac.dover@proton.me> <1445107+zdover23@users.noreply.github.com>
+Zac Dover <zac.dover@proton.me> <zac.dover@gmail.com>
 Zack Cerza <zack@redhat.com> <zack.cerza@inktank.com>
 Zack Cerza <zack@redhat.com> <zack@cerza.org>
 Zack Cerza <zack@redhat.com> <zcerza@redhat.com>
+Zack Cerza <zack@redhat.com> zmc <NOT@FOUND>
```

Comment:
I'm unsure what this line should contain, but the relevant email addresses here are Red Hat and cerza[.]org.

Reply:
ahhh thank you for that!

I wonder, then, if removing the inktank address is the right thing to do

Reply:
your inktank address wasn't present in any commits between quincy..reef so it's not needed for the reef contributor credits, but there's no harm in keeping it in there; up to you

Reply:
Ah, got it - let's leave it out then. Thank you!